Angular/src/app/data.service.ts

```diff
@@ -83,4 +83,44 @@ export class DataService {
     });
   }
 
```

Comment:
These methods will be updated over the weekend to actually implement reading from PO files

Reply:
I think the PO files on the front end shouldn't have to make requests to the RESTful API. I was thinking the default values for the strings would be English, but that's just because that's the common language we're writing in currently.